src/components/OrgFile/components/TaskListModal/components/TaskListView/index.js

```diff
@@ -101,11 +101,15 @@ function TaskListView(props) {
         return [earliestPlanningItem, header];
       })
       .sortBy(([planningItem, header]) => {
+        console.log(todoKeywordSets);
+        const doneState = todoKeywordSets
```

Comment:
Could this be done with the `createIsTodoKeywordInDoneState` in parse_utils.js?

Reply:
I propose we add a test to document the functionality. Refactoring can be done with more certainty, too.

I'll add at least one.

Reply:
Right, should be possible to use that function. I thought I considered that... I probably did not use it because the documentation of this function was misleading. It has a mistake in the second line
...
if said `todoKeyword` is in `todoKeywordSets`.

I think it should be "if said `todoKeyword` is in any `todoKeywordSets`s done states."

Reply:
The test revealed that the current sort order for the `testOrgFile` in the integration test is:

1. A todo item with schedule and deadline
1. A header with a custom todo sequence in DONE state
1. A headline that's done a day earlier even
1. A headline that's done since a loong time

It's the same after using the function from `org_utils`, so I feel confident we can use it(;

> I think it should be "if said todoKeyword is in any todoKeywordSetss done states."

Right! I chaged it, thanks for the pointer!